strawberryfields/backends/tfbackend/circuit.py

```diff
@@ -612,161 +551,147 @@ def measure_fock(self, modes, select=None, **kwargs):
                     raise ValueError("'select' must be have the same shape as 'modes'")
 
         # carry out the operation
-        with self.graph.as_default():
-            evaluate_results, session, feed_dict, close_session = ops._check_for_eval(kwargs)
```

Comment:
Similar question to `graph`: how come `session`, `eval` and `feed_dict` were removed? (For my own understanding)

Reply:
Same reason; having separate sessions, evaluations, and feed dictionaries were part of graph mode and thus not relevant to TensorFlow 2.0